docs/src/_reference/command-line-interface.md

```diff
@@ -701,16 +701,18 @@ Note that if no environment is active, `meltano run` _does not_ generate a state
 
 ## `job`
 
-Use the `job` command to define one or more related tasks. A job can contain a single task, or many tasks. Today all tasks
-are run sequentially. You can run a specified job by passing the job name as an argument to `meltano run`. You can also schedule jobs using `meltano schedule`.
+Use the `job` command to define one or more sequences of tasks. A job can contain a single task or many tasks.
+As of today all tasks are run sequentially.
+You can run a specified job by passing the job name as an argument to [`meltano run`](#run).
+You can also schedule jobs using [`meltano schedule`](#schedule).
 
 ### How to use
 
 ```bash
 # Add a job with a single task representing a run command
 meltano job add <job_name> --tasks "<tap_name> <mapping_name> <target_name> <command>"
 
-# Add a new job with multiple tasks by passing arrays in yaml format, where each item representing a run command.
+# Add a new job with multiple tasks by passing arrays in yaml format, where each item represents a run command.
 meltano job add <job_name> --tasks "[<tap_name> <target_name>, <command>, <tap2_name> <target2_name>, ...]"
```

Comment:
Should we illustrate how this breaks down into tasks? I think thats what we did in the cli reference and `--help` usage too. Something like:

```
# Add a job with a single task representing a single run command
meltano job add <job_name> --tasks "<tap_name> <mapping_name> <target_name> <command>"

# Add a job with multiple tasks
# task1: tap_name target_name
# task2: command
# task3: tap2_name target2_name
meltano job add <job_name> --tasks "[<tap_name> <target_name>, <command>, <tap2_name> <target2_name>, ...]"
```